exp/containers.md

```diff
@@ -26,28 +26,36 @@ All containers in Anbox Cloud are ephemeral, which means that as soon as a conta
 
 ## Container life cycle
 
-When you create a container by launching an application or an image, it executes the following steps in order:
+When you [create a container](https://discourse.ubuntu.com/t/launch-a-container/24327) by either launching or initialising an application or an image, it executes the following steps in order:
 
+1. Schedule a container on LXD node.
 1. Configure the network interface and gateway.
-2. Only raw containers: Install addons that are specified with `--addons`.
-3. Expose services that are specified with `--service` or through the application manifest.
-4. Execute the `pre-start` hook provided by the installed addons.
-5. Launch the Android container.
-6. Execute the `post-start` hook provided by the installed addons.
+1. Only raw containers: Install addons that are specified with `--addons`.
+1. Expose services that are specified with `--service` or through the application manifest.
+1. Execute the `pre-start` hook provided by the installed addons.
+1. Launch the Android container.
+1. Execute the `post-start` hook provided by the installed addons.
 
-![Container start|566x528](https://assets.ubuntu.com/v1/0f386949-container_start.png)
+![Container start|584x646](TBD)
 
 The whole launch process is successful only if all of the above steps succeed.
 
 If anything goes wrong during the container launch process, the status of the container changes to the `error` status. You can [view the available logs](https://discourse.ubuntu.com/t/view-the-container-logs/24329) from the container for diagnosing the root cause of the problem.
 
-When a container is stopped, either because you deleted it or because an error occurred, it executes the following steps in order:
+When a container is stopped, because one of the following scenarios occurs:
+
+- you stopped it.
+- you deleted it.
+- an error occurred.
+
+it executes the following steps in order:
```

Comment:
We shouldn't have a list like this in the middle of a sentence.
How about:
```suggestion
Containers can be stopped because of the following scenarios:

- You stopped it.
- You deleted it.
- An error occurred.

When a container is stopped, it executes the following steps in order:
```